scripts/variantstore/AOU_DELIVERABLES.md

```diff
@@ -1,40 +1,84 @@
-    3. the default_dataset you used for all the GVS WDLs
-    4. the `extract_table_prefix` input from `GvsExtractCallset` step
-    5. the `filter_set_name` input from `GvsCreateFilterSet` step
-
-## Deliverables (via email once the above steps are complete)
-1. location of the VCFs and interval_list files (`output_gcs_dir` input from GvsExtractCallset)
-2. fully qualified name of the BigQuery dataset (input `dataset_name` in the workflows)
-3. [callset statistics](callset_QC/README.md) CSV
-4. [precision and sensitivity results](tieout/AoU_PRECISION_SENSITIVITY.md) TSV
+## Setup
+- Create a Terra workspace
+  - using your @pmi-ops.org account 
+  - with the `allofus-drc-wgs-dev` Terra Billing account 
+  - in the `AOU_DRC_WGS_OPERATORS` Authorization Domain
+  - Share it with `dsp-variant-team@firecloud.org` so the rest of the team can see it (Reader) and with the individual users who will be running the workflows (Owner).
+- Populate the workspace with the following:
+  - [Fetch WGS metadata for samples from list](http://app.terra.bio/#workspaces/allofus-drc-wgs-dev/GVS%20AoU%20WGS%20Charlie/notebooks/launch/Fetch%20WGS%20metadata%20for%20samples%20from%20list.ipynb) notebook
+  - [GvsAssignIds](https://dockstore.org/my-workflows/github.com/broadinstitute/gatk/GvsAssignIds) workflow
+  - [GvsImportGenomes](https://dockstore.org/my-workflows/github.com/broadinstitute/gatk/GvsImportGenomes) workflow
+  - [GvsWithdrawSamples](https://dockstore.org/my-workflows/github.com/broadinstitute/gatk/GvsWithdrawSamples) workflow
+  - [GvsCreateAltAllele](https://dockstore.org/my-workflows/github.com/broadinstitute/gatk/GvsCreateAltAllele) workflow
+  - [GvsCreateFilterSet](https://dockstore.org/my-workflows/github.com/broadinstitute/gatk/GvsCreateFilterSet) workflow
+  - [GvsPrepareRangesCallset](https://dockstore.org/my-workflows/github.com/broadinstitute/gatk/GvsPrepareRangesCallset) workflow (VCF output)
+  - [GvsExtractCallset](https://dockstore.org/my-workflows/github.com/broadinstitute/gatk/GvsExtractCallset) workflow (VCF output)
+  - [GvsCalculatePrecisionAndSensitivity](https://dockstore.org/workflows/github.com/broadinstitute/gatk/GvsCalculatePrecisionAndSensitivity) workflow
+  - **TBD VDS Prepare WDL/notebook/??**
+  - **TBD VDS Extract WDL/notebook/??**
+- Run the "Fetch WGS metadata for samples from list" notebook after you have placed the file with the list of the new samples to ingest in a GCS location the notebook (running with your @pmi-ops account) will have access to.  This will grab the samples from the workspace where they were reblocked and bring them into this callset workspace.
+  - Set the `sample_list_file_path` variable in that notebook to the path of the file
+  - Run the "now that the data have been copied, you can make sample sets if you wish" step if you want to automatically break up the new samples into smaller sample sets.  Set the `SUBSET_SIZE` and `set_name` variables to customize.
```

Comment:
Under what circumstances would we want to do this?

Reply:
If you don't want to throw all 100K (or whatever number) samples at `GvsImportGenomes` at once.